compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -2485,41 +2509,53 @@ LogicalResult AttentionOp::verify() {
     size_t lastIdx = valueShape.size() - 1;
     std::swap(valueShape[lastIdx - 1], valueShape[lastIdx]);
   }
-  if (failed(verifyCompatibleShape(keyShape, valueShape)))
+  if (failed(verifyCompatibleShape(keyShape, valueShape))) {
     return op->emitOpError("incompatible value shape");
-  if (failed(verifyCompatibleShape(queryShape, outputShape)))
+  }
+  if (failed(verifyCompatibleShape(queryShape, outputShape))) {
     return op->emitOpError("incompatible output shape");
-  if (queryElementType != keyElementType || keyElementType != valueElementType)
+  }
+  if (queryElementType != keyElementType ||
+      keyElementType != valueElementType) {
     return op->emitOpError(
         "element types of (Q)uery, (K)ey and (V)value should be same");
+  }
   if (numOperands == 4) {
     // Vanilla attention.
-    if (queryElementType != outputElementType)
+    if (queryElementType != outputElementType) {
       return op->emitOpError("expected element type for Output ")
              << queryElementType << "but found " << outputElementType
              << " instead";
-    if (keyShape[2] != queryShape[2])
+    }
+    if (keyShape[2] != queryShape[2]) {
       return op->emitOpError("query and key head dimension mismatch");
+    }
   }
   if (numOperands == 6) {
     // Tiled/Flash attention.
     ShapedType maxType = *getMaxType();
     ShapedType sumType = *getSumType();
-    if (failed(checkShapeRank(op, "max", maxType, 1)))
+    if (failed(checkShapeRank(op, "max", maxType, 1))) {
       return failure();
```

Comment:
For some of these diffs, I'm wondering if an in-place cleanup PR (modifying the files while they are still in their old location) would be easier to review. As this PR is, the diffs are split across many commits.

Reply:
I was expecting reviewers to take a look at https://github.com/openxla/iree/pull/16407/commits/4f8cf66948192f4d96e5110f8ec37fa65ef23551

Dealing these with Github is very painful, so I put several commits in the PR.